### PR TITLE
Add preferences to User resource

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: spec/definitions/User.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@ properties:
     type: string
   preferences:
     description: User preferences like timezone, language and many more. This is an object with custom properties.
+    type: object
```
Comment (slavcodev) [inline spec/definitions/User.yaml]: Hmm, is not that preference pre-defined, is it?

Comment (XaosSintez) [inline spec/definitions/User.yaml]: They are not pre defined

Comment (slavcodev) [inline spec/definitions/User.yaml]: Wonder how they can be variable if client (at least our UI) requires specific ones?

Comment (XaosSintez) [inline spec/definitions/User.yaml]: They don't require it. They set it and use it. Actually, backend has no idea, which properties will be there and backend doesn't use it at all.
It is totally defined by FE and used only by FE

Comment (slavcodev) [inline spec/definitions/User.yaml]: Alright.

**Pull request merged**